builtins/amp-ad.md

```diff
@@ -26,7 +26,60 @@ limitations under the License.
 
 AMP documents only support ads served via HTTPS.
 
-#### Behavior
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A container to display an ad.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><a href="https://github.com/ampproject/amphtml/blob/master/examples/ads.amp.html">ads.amp.html</a></td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-ad` tag
+(see also `amp-ad` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
+
+<table>
```

Comment:
"Normal" tables like this can be done in native markdown:

```md
| Header 1 | Header 2 |
| -------- | -------- |
| row 1, column 1 | row 1 column 2 |
| row 2, column 1 | row 2 column 2 |
```

| Header 1 | Header 2 |
| -------- | -------- |
| row 1, column 1 | row 1 column 2 |
| row 2, column 1 | row 2 column 2 |